pipeline_config/pipeline-kyc-id.yaml

```diff
@@ -0,0 +1,102 @@
+#   -t argus.processors.ocr_processors.Intake root_docs_path=5pdfs follow_symlinks=true \
+#   -t argus.processors.ocr_processors.PdfTextExtract \
+#   -t argus.processors.ocr_processors.NormalizeImages resample_to_dpi=300 normalize_image_format=.png \
+#   -t argus.processors.ocr_processors.GenericOcr ocr_style=TesseractOcr \
+#   --reorder_inputs None,0 \
+#   -t argus.processors.train_eval_processors.TrainEvalProcessor base_model_name_or_path=/PathToModel \
+#   --reorder_inputs None,0 \
+#   -t argus.processors.train_eval_processors.TrainEvalProcessor base_model_name_or_path=/PathToModel \
+#   -t argus.processors.post_processors.generic_post_processor.PostProcessor \
+#   -o 5pdfs_scoring_in_one_go
+
+
+  pipeline:
+    steps:
+    - tasks:
+       - name: "Intake" #name of task
+         type: PipelineTask # if absent, defaults to PipelineTask.  Can also be PipelineReorderInputs and InputCommand.  InputCommand cannot be used in the scorer.
+         class: argus.processors.ocr_processors.Intake # fqn_of_Processor class
+         parameters:
+           root_docs_path: /input/path
+           follow_symlinks: true
+    - tasks:
+       - name: "PdfExtract" #name of task
+         type: PipelineTask # if absent, defaults to PipelineTask.  Can also be PipelineReorderInputs and InputCommand.  InputCommand cannot be used in the scorer.
+         class: argus.processors.ocr_processors.PdfTextExtract # fqn_of_Processor class
+    - tasks:
+       - name: "ImageNormalize" #name of task
+         class: argus.processors.ocr_processors.NormalizeImages
+         parameters:
+           resample_to_dpi: 300 
+           normalize_image_format: .jpg
+    - tasks:
+       - name: "OCR" #name of task
+         class: argus.processors.ocr_processors.GenericOcr
+         parameters:
+           ocr_style: E3DocTROcr
+    - tasks:
+       - name: "move input annoset from 0th into 1st position"
+         type: PipelineReorderInputs
+         reordering: [null, 0]
+    - tasks:
+       - name: "Predict"
+         class: argus.processors.train_eval_processors.TrainEvalProcessor
+         parameters:
+           artifacts:
+              base_model_name_or_path: id_card_model #matches the basename before .zip
+    - tasks:
+       - name: "move input annoset from 0th into 1st position"
+         type: PipelineReorderInputs
+         reordering: [null, 0]
+    - tasks:
+       - name: "Predict" # Are we allow a duplicated tasks name? or have to be one as token predict and another as page predict
+         class: argus.processors.train_eval_processors.TrainEvalProcessor
+         parameters:
+           uses_resource_dir: false
+           artifacts:
+              base_model_name_or_path: page_class_model #matches the basename before .zip
+    - tasks:
+       - name: "PostProcess"
+         class: argus.processors.post_processors.generic_post_processor.PostProcessor
+         parameters:
+           output_format: 'json'
+           labeling_threshold: 0.8
```

Comment:
I would recommend using 0.2 as the default threshold for the token labeling model. Because after token merging and probability averaging, some entities get pretty low confidences, and they are correct.